app/code/Magento/InventoryInStorePickupQuote/Plugin/Quote/Address/SetShippingDescription.php

```diff
@@ -0,0 +1,117 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickupQuote\Plugin\Quote\Address;
+
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\InventoryInStorePickup\Model\GetPickupLocation;
```

Comment:
Please use abstraction instead of implementation.

Reply:
ok